style.css

```diff
@@ -0,0 +1,947 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  background-color: #f7f7f9;
+  width: 100%;
+  height: 6500px;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.introduction {
+  width: 23.5rem;
+  height: 735px;
+  background: #fff;
+  margin-top: 131px;
+  border-bottom-left-radius: 100px;
+  margin-bottom: 80px;
+  background-image: url(./images/background.png);
+  background-repeat: no-repeat;
+  background-position-y: 100px;
+}
```

Comment:
Kindly, I recommend you to use flexbox to adapt the information to the center of the container